include/ofi.h

```diff
@@ -326,6 +326,16 @@ static inline void *ofi_get_page_end(const void *addr, size_t page_size)
 			+ page_size, page_size) - 1);
 }
 
+static inline bool ofi_is_size_aligned(size_t size, size_t alignment)
+{
+	return ((size % alignment) == 0);
```

Comment:
given alignment is always power of 2, maybe (size & (alignment -1) ==0) would be more efficient?

Reply:
I see all the size alignment checks in this file are using `%`. I wonder if there is a reason. @shefty

Reply:
Do we know that alignment and multiple values are always a value of 2?  We can enforce that through assert(), but it's currently not a requirement.

As long as the input values are either constants or use sizeof(), a smart compiler should be able to identify this and generate an optimal check.  Though, the use of sizeof() suggests that a power of 2 isn't likely.